website/react-magma-docs/src/pages/api/aibutton.mdx

```diff
@@ -0,0 +1,315 @@
+Because of the problems that animation can create for people who are sensitive to such things,
+we automatically disable it in the AI Button if we detect that the `prefers-reduced-motion` media query is set.
+
+```tsx
+import React from 'react';
+
+import { AIButton, AIButtonVariant, ButtonGroup } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <ButtonGroup>
+      <AIButton isAnimated>Animated Gradient</AIButton>
+      <AIButton isAnimated variant={AIButtonVariant.variantB}>
+        Animated Second Gradient
+      </AIButton>
+      <AIButton isAnimated leftColor="#f00" rightColor="#0f0">
+        Animated Custom Gradient
+      </AIButton>
+    </ButtonGroup>
+  );
+}
+```
+
+## Custom Colors
+
+You can override gradient colors using `leftColor`, `rightColor`, `hoverColor`, and `pressedColor`.
+
+```tsx
+import React from 'react';
+
+import { AIButton } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <AIButton
+      leftColor="#00f"
+      rightColor="#0ff"
+      hoverColor="#0f0"
+      pressedColor="#f00"
+    >
+      Custom Colors
+    </AIButton>
+  );
+}
+```
+
+## Loading State
+
+Set `isLoading` to show a spinner and disable the button.
+
+```tsx
+import React from 'react';
+
+import { AIButton, ButtonGroup } from 'react-magma-dom';
+
+export function Example() {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [isLoadingIcon, setIsLoadingIcon] = React.useState(false);
+
+  React.useEffect(() => {
+    if (isLoading) {
+      setTimeout(() => {
+        setIsLoading(false);
+      }, 4000);
+    }
+
+    if (isLoadingIcon) {
+      setTimeout(() => {
+        setIsLoadingIcon(false);
+      }, 4000);
+    }
+  }, [isLoading, isLoadingIcon]);
+
+  return (
+    <ButtonGroup>
+      <AIButton isLoading={isLoading} onClick={() => setIsLoading(true)}>
+        Loading Button
+      </AIButton>
+      <AIButton
+        isLoading={isLoadingIcon}
+        onClick={() => setIsLoadingIcon(true)}
+      />
+    </ButtonGroup>
+  );
+}
+```
+
+## Disabled State
+
+Set `disabled` to disable the button.
+
+```tsx
+import React from 'react';
+
+import { AIButton } from 'react-magma-dom';
+
+export function Example() {
+  return <AIButton disabled>Disabled</AIButton>;
+}
+```
+
+## Inverse
+
+Use `isInverse` for dark backgrounds.
+
+```tsx
+import React from 'react';
+
+import { AIButton, Card, CardBody } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse>
+      <CardBody>
+        <AIButton isInverse>Inverse</AIButton>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
+## Event Handling
+
+The `onClick` event fires when the button is clicked or activated by keyboard.
+
+```tsx
+import React from 'react';
+
+import { AIButton } from 'react-magma-dom';
+
+export function Example() {
+  return <AIButton onClick={() => alert('Clicked!')}>Click Me</AIButton>;
+}
```

Comment:
We should update this example to match the one in Button with a counter-- alerts are annoying :P